clippy_lints/src/doc.rs

```diff
@@ -413,6 +414,16 @@ fn check_doc<'a, Events: Iterator<Item = (pulldown_cmark::Event<'a>, Range<usize
                 };
                 let (begin, span) = spans[index];
                 if in_code {
+                    let lo = span.lo() + BytePos::from_usize(range.start - begin);
+                    // we need a span that includes at least the code.
+                    // however, because we have already stripped the comment
+                    // prefixes, we need to add their size back. We currently
```

Comment:
```suggestion
                    // prefixes (`/// `), we need to add their size back. We currently
```

Took me way too long to figure out what you meant by "comment prefixes".  Can you add this for people like me? 😄

Reply:
Sure. I'm rebuilding the branch anyway, because somehow a rebase went wrong.